src/test/groovy/com/causecode/fileuploader/UFileMoveHistorySpec.groovy

```diff
@@ -7,15 +7,14 @@
  */
 package com.causecode.fileuploader
 
-import grails.test.mixin.TestFor
+import grails.testing.gorm.DomainUnitTest
 import spock.lang.Specification
 import spock.lang.Unroll
 
 /**
  * This is unit test file for UFileMoveHistory class.
  */
-@TestFor(UFileMoveHistory)
-class UFileMoveHistorySpec extends Specification implements BaseTestSetup {
+class UFileMoveHistorySpec extends Specification implements DomainUnitTest<UFileMoveHistory>, BaseTestSetup {
```

Comment:
DomainUnitTest ->  BuildDomainTest, According to current version 3.3.0 Link: http://longwa.github.io/build-test-data/index

Reply:
I am using https://testing.grails.org/latest/api/grails/testing/gorm/DomainUnitTest.html
test-data plugin's classes are not required here since we are not creating any test instance in this test file.